### PR TITLE
adding category, parameters and types of level for #issue196

### PR DESCRIPTION
**Pull request by amilan17**
#196

### CODE REVIEW

**Review by amilan17 — COMMENTED** [review]
two comments and waiting for note on deprecated parameters

File: GRIB2_CodeFlag_4_2_CodeTable_en.csv [hunk 1]
```diff
@@ -646,7 +660,8 @@ Parameter number by product discipline and parameter category,"Product disciplin
 Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 21: thermodynamic properties",19,,Column integrated eastward heat flux,,,W m-1,Operational
 Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 21: thermodynamic properties",20,,Column integrated northward heat flux,,,W m-1,Operational
 Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 21: thermodynamic properties",21,,Column integrated potential+internal+latent energy,,,J m-2,Operational
-Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 21: thermodynamic properties",22-191,,Reserved,,,,Operational
+Parameter number by product discipline and parameter category,"Product discipline 0 - Meteorological products, parameter category 21: thermodynamic properties",22,,Eady growth rate,,,numeric,Operational
```
[the remark this answers is not in the document]
Comment (sebvi) [reply GRIB2_CodeFlag_4_2_CodeTable_en.csv:663]: correct, changed to day-1 (although the real SI units would be s-1 but then most values would be extremely small, in the order of 10-6 s-1)

File: GRIB2_CodeFlag_4_2_CodeTable_en.csv [hunk 2]
```diff
@@ -888,7 +910,9 @@ Parameter number by product discipline and parameter category,"Product disciplin
 Parameter number by product discipline and parameter category,"Product discipline 3 - Space products, parameter category 1: quantitative products",28,,Brightness temperature,,,K,Operational
 Parameter number by product discipline and parameter category,"Product discipline 3 - Space products, parameter category 1: quantitative products",29,,Scaled radiance,(see Note 2),114,Numeric,Operational
 Parameter number by product discipline and parameter category,"Product discipline 3 - Space products, parameter category 1: quantitative products",30,,Reflectance in 0.4 micron channel,,,%,Operational
-Parameter number by product discipline and parameter category,"Product discipline 3 - Space products, parameter category 1: quantitative products",31-97,,Reserved,,,,Operational
```
[the remark this answers is not in the document]
Comment (sebvi) [reply GRIB2_CodeFlag_4_2_CodeTable_en.csv:891]: just realised I forgot to put the discipline 3, category 1 table in the proposal. I am adding it now

File: notes/CodeFlag_notes.csv [hunk 3]
```diff
@@ -133,4 +133,4 @@ noteID,note
 136,"Confidence interval for the significance test is needed (NA=1)."
 137,"EFI is defined in https://doi.org/10.1256/qj.02.152. No additional parameter needed (NA=0)."
 138,"SOT is defined in https://doi.org/10.1256/qj.02.152. Lower and upper quantile used for the SOT need to be defined (NA=2) in this order: start of the tail (10 or 90 usually) then end of tail (1 or 99 respectively)."
-
+iss196a,"Parameters are deprecated. It is recommended to use 63 and 64 instead."
```
Comment (amilan17) [inline notes/CodeFlag_notes.csv:136]: @sebvi and @marijanacrepulja I added the note. Can you review it?

**Comment by amilan17** [comment]
Branch is ready, except (maybe) the note.

**Pull request merged**